modules/observe/proc-forward-audit-log-splunk.adoc

```diff
@@ -0,0 +1,187 @@
+[source,yaml]
+----
+serviceAccount:
+  name: log-collector
+----
+--
+.. Configure `inputs` to specify the type and source of logs to forward. The following configuration enables the forwarder to capture logs from all applications in a provided namespace:
++
+--
+.Example `inputs` configuration
+[source,yaml]
+----
+inputs:
+  - name: my-app-logs-input
+    type: application
+    application:
+      includes:
+        - namespace: my-developer-hub-namespace
+      containerLimit:
+        maxRecordsPerSecond: 100
+----
+
+For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html-single/logging/index#cluster-logging-collector-log-forward-logs-from-application-pods_configuring-log-forwarding[Forwarding application logs from specific pods].
+--
+.. Configure outputs to specify where the captured logs are sent. In this step, focus on the `splunk` type. You can either use `tls.insecureSkipVerify` option if the Splunk endpoint uses self-signed TLS certificates (not recommended) or provide the certificate chain using a Secret.
++
+--
+.Example `outputs` configuration
+[source,yaml]
+----
+outputs:
+  - name: splunk-receiver-application
+    type: splunk
+    splunk:
+      authentication:
+        token:
+          key: hecToken
+          secretName: splunk-secret
+      index: main
+      url: 'https://my-splunk-instance-url'
+      rateLimit:
+        maxRecordsPerSecond: 250
+----
+
+For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html-single/logging/index#logging-forward-splunk_configuring-log-forwarding[Forwarding logs to Splunk] in {ocp-short} documentation. 
+--
+.. Optional: Filter logs to include only audit logs:
++
+--
+.Example `filters` configuration
+[source,yaml]
+----
+filters:
+  - name: audit-logs-only
+    type: drop
+    drop:
+      - test:
+        - field: .message
+          notMatches: isAuditLog
+----
+For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html-single/logging/index#logging-content-filtering[Filtering logs by content] in {ocp-short} documentation.
+--
+.. Configure pipelines to route logs from specific inputs to designated outputs. Use the names of the defined inputs and outputs to specify multiple `inputRefs` and `outputRefs` in each pipeline:
++
+--
+.Example `pipelines` configuration
+[source,yaml]
+----
+pipelines:
+  - name: my-app-logs-pipeline
+    detectMultilineErrors: true
+    inputRefs:
+      - my-app-logs-input
+    outputRefs:
+      - splunk-receiver-application
+    filterRefs:
+      - audit-logs-only
+----
+--
+
+. Run the following command to apply the `ClusterLogForwarder` configuration:
++
+--
+.Example command to apply `ClusterLogForwarder` configuration
+[source,bash]
+----
+oc apply -f <ClusterLogForwarder-configuration.yaml>
+----
+--
+. Optional: Customize your `ClusterLogForwarder` pods using the following options:
```

Comment:
It would be nice if it stated what the `default` tuning options are if you leave it blank, but I don't see anything about it in the docs ... https://docs.openshift.com/container-platform/4.14/observability/logging/log_collection_forwarding/configuring-log-forwarding.html#logging-delivery-tuning_configuring-log-forwarding

Reply:
Ok, I found it, the `AtLeastOnce` option is the default, https://github.com/openshift/cluster-logging-operator/blob/master/docs/features/collection.adoc#tuning

Reply:
Done